From: Jan Engelhardt <jengelh@medozas.de>

build: fix compiler warnings

oplplayer.c:1347:9: warning: format '%i' expects type 'int', but
argument 3 has type 'size_t'
gl_main.c:1307:5: warning: format '%i' expects type 'int', but
argument 3 has type 'long int'

size_t requires %z; the second instance actually has type ptrdiff_t,
so %t is required.

---
 src/MUSIC/oplplayer.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: prboom-plus-2.5.1.3/src/MUSIC/oplplayer.c
===================================================================
--- prboom-plus-2.5.1.3.orig/src/MUSIC/oplplayer.c
+++ prboom-plus-2.5.1.3/src/MUSIC/oplplayer.c
@@ -1356,7 +1356,7 @@ static const void *I_OPL_RegisterSong(co
     // time numbers we have to traverse the tracks and everything
     if (mf.len < 100)
     {
-        lprintf (LO_WARN, "I_OPL_RegisterSong: Very short MIDI (%i bytes)\n", mf.len);
+        lprintf (LO_WARN, "I_OPL_RegisterSong: Very short MIDI (%zu bytes)\n", mf.len);
         return NULL;
     }
 
